### PR TITLE
FIX - Update physical network traffic labels

### LINKED ISSUE #882: [BUG] When editing physical network traffic labels - existing labels are not shown
image

When you choose a kind of traffic - the existing labels should be shown, as in old UI (and it is a common sense, IMO to show existing values, and not just empty fields)

Old UI below:

image

### PR DESCRIPTION
**Pull request by utchoang**
Fixes #882 
@rhtyd cc @svenvogel Please review it. Thanks.
image

### CODE REVIEW

**Comment by utchoang** [comment]
@blueorangutan package

File: src/views/infra/network/EditTrafficLabel.vue [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        <a-form-item>
+          <span slot="label">
+            {{ $t('label.kvmnetworklabel') }}
+            <a-tooltip :title="apiParams.kvmnetworklabel.description">
+              <a-icon type="info-circle" style="color: rgba(0,0,0,.45)" />
+            </a-tooltip>
+          </span>
+          <a-input
+            v-decorator="['kvmnetworklabel', {
+              initialValue: trafficResource.kvmnetworklabel || $t('label.network.label.display.for.blank.value')
```
Comment (davidjumani) [inline src/views/infra/network/EditTrafficLabel.vue:50]: @utchoang Would it be better to use `$t('label.network.label.display.for.blank.value')` as a placeholder instead?
Also the user is unable to clear or reset the tag once it's been set

**Comment by utchoang** [comment]
@blueorangutan package

File: src/views/infra/network/EditTrafficLabel.vue [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    handleSubmit (e) {
+      e.preventDefault()
+      this.form.validateFields((err, values) => {
+        if (err) {
+          return
+        }
+        this.loading = true
+        const params = {}
+        for (const key in values) {
+          params[key] = values[key] ? values[key].trim() : undefined
```
Comment (davidjumani) [inline src/views/infra/network/EditTrafficLabel.vue:185]: @utchoang Unable to reset the traffic label. Think removing this check might fix it

**Comment by utchoang** [comment]
@blueorangutan package

File: src/views/infra/network/EditTrafficLabel.vue [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <div class="form-layout">
+    <a-spin :spinning="loading">
+      <a-form :form="form" :loading="loading" @submit="handleSubmit" layout="vertical">
+        <a-form-item>
+          <span slot="label">
+            {{ $t('label.id') }}
```
Comment (davidjumani) [inline src/views/infra/network/EditTrafficLabel.vue:24]:
```suggestion
            {{ $t('label.traffictype') }}
```

**Review by davidjumani — APPROVED** [review]
Minor suggestion. Apart from that LGTM!
Thanks @utchoang

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by rhtyd — APPROVED** [review]

**Review by rhtyd — APPROVED** [review]

**Comment by rhtyd** [comment]
LGTM, tested few cases with different traffic types. Existing values seen populated in the update form:
Screenshot from 2020-12-03 12-27-29

**Pull request merged**